Makefile

```diff
@@ -12,6 +12,7 @@ clean:
 	rm -f ./build.log
 
 package-install:
+	npm audit --audit-level=moderate
```

Comment:
We have an npm config file (`.npmrc`) which has the audit-level set to moderate.
Will the CI not pick that up, so we can just do `npm audit` in the Makefile instead?

Reply:
You are right, i'll remove from here to avoid duplication.